docs/contributing.md

```diff
@@ -44,11 +44,11 @@ Pull requests containing tabs will not be accepted. Make sure you set your edito
 
 ## Line Endings
 
-The repository is configured to preserve line endings both on checkout and commit (the equivalent of `autocrlf` set to `false`). This means *you* are responsible for line endings. We recommend that you configure your diff viewer so that it does not ignore line endings. Any [wall of pink](https://www.hanselman.com/blog/YoureJustAnotherCarriageReturnLineFeedInTheWall.aspx) pull requests will not be accepted.
+The repository is configured to preserve line endings both on checkout and commit (the equivalent of `autocrlf` set to `false`). This means *you* are responsible for line endings. We recommend that you configure your diff viewer so that it does not ignore line endings. Any [wall of pink](https://www.hanselman.com/blog/youre-just-another-carriage-return-line-feed-in-the-wall) pull requests will not be accepted.
 
 ## Line Width
 
-Try to keep lines of code no longer than 160 characters wide. This isn't a strict rule. Occasionally a line of code can be more readable if allowed to spill over slightly. A good way to remember this rule is to use the 'Column Guides' feature of the [Productivity Power Tools 2012](https://visualstudiogallery.msdn.microsoft.com/3a96a4dc-ba9c-4589-92c5-640e07332afd) extension for Visual Studio.

```

Comment:
I don't know if this extension still exists, but if it does, it's so old it probably doesn't make sense to mention it here. Pretty sure VS can do that without an extension now? The link is broken anyway, the certificate is expired.

Reply:
There's a [Productivity Power Tools 2022](https://marketplace.visualstudio.com/items?itemName=VisualStudioPlatformTeam.ProductivityPowerPack2022), but it doesn't mention column guides. Pretty sure it's built into VS now.